weaver/wps_restapi/jobs/jobs.py

```diff
@@ -38,7 +38,30 @@
 LOGGER = get_task_logger(__name__)
 
 
-def check_status(url=None, response=None, sleep_secs=2, verify=False, settings=None):
+def get_local_status_location(url_status_location, container, must_exist=True):
+    # type: (AnyStr, AnySettingsContainer, bool) -> Optional[AnyStr]
+    """Attempts to retrieve the local file path corresponding to the WPS status location as URL.
+
+    :param url_status_location: URL reference pointing to some WPS status location XML.
+    :param container: any settings container to map configured local paths.
+    :param must_exist: return only existing path if enabled, otherwise return the parsed value without validation.
+    :returns: found local file path if it exists, ``None`` otherwise.
+    """
+    if url_status_location and not urlparse(url_status_location).scheme in ["", "file://"]:
+        dir_path = get_wps_output_dir(container)
+        wps_out_url = get_wps_output_url(container)
+        req_out_url = get_url_without_query(url_status_location)
+        out_path = os.path.join(dir_path, req_out_url.replace(wps_out_url, "").lstrip("/"))
+    else:
+        out_path = url_status_location.replace("file:://", "")
```

Comment:
typo in file::// (double ::) or this is really a case?

Reply:
Intentional thing that I found a while ago. I don't recall 100% the use case... should have commented it back then. I think is was related to pywps doing this modification. Maybe it is not the case anymore, I have not seen this else branch being called during my recent tests.